src/resources/views/columns/permissions.blade.php

```diff
@@ -0,0 +1,51 @@
+{{-- relationships with pivot table (n-n) --}}
+<td>
+    <?php
+
+    // Gets the attribute to display the permission name
+    $attribute = array_get($column, 'attribute');
+
+    $permissions = $entry->{$column['entity']};
+
+    // Groups permissions by prefix, sorted alphabetically
+    $permissionsByPrefix = collect($permissions)
+        ->sortBy(function($permission) {
+            return $permission->prefix() ?: PHP_INT_MAX; // Use PHP_INT_MAX as a little trick for sorting permissions without prefix at the end
```

Comment:
Ok, "alphabetical" and PHP_INT_MAX cause me to have mental dissonance.

What is that return comparing against (or is there some link explaining why it works)?